nvtabular/column_selector.py

```diff
@@ -40,6 +40,12 @@ def grouped_names(self):
             names.append(tuple(subgroup.names))
         return names
 
+    def __nested_check(self, nests=0):
+        if nests > 2:
+            raise AttributeError("Too many nested subgroups")
+        for col_sel0 in self.subgroups:
+            col_sel0.__nested_check(nests=nests + 1)
```

Comment:
I'm not entirely sure, but there might be an off by one error here? We start with `nests=0`, so the first nested layer has `nests=1` and that's the maximum nesting we allow (I think.)

Reply:
oh man my bad. I thought we were running 2 levels deep... I didn't count start level.

Reply:
No worries! It's so common and easy to do in this field. We have jokes about it for a reason. 😃